test/performance/latency/latency-part1.go

```diff
@@ -34,7 +34,7 @@ var (
 	BINS         int    = 10
 	SKIP_NUMBER  uint64 = 100000
 	SPEED        uint64 = 1000000
-	PASSED_LIMIT uint64 = 90
```

Comment:
Yes we can achieve higher rate 84-91%. To do that
1. generated packets should be diverse
2. sent packet number should be increased (total sent ~10 mln)
I will update test now